source/_docs/installation/synology.markdown

```diff
@@ -2,78 +2,287 @@
+* Install Home Assistant
+* Enable Home Assistant to be launched on http://*Synology_IP*:8123
+
+Using the [Synology webadmin](https://www.synology.com/en-global/knowledgebase/DSM):
+* Install compiled Python3 package using the [Synology Package Center](https://www.synology.com/en-global/knowledgebase/DSM/help/DSM/PkgManApp/PackageCenter_desc)
+* Create “homeassistant” user and add to the “users” group
+* Create “homeassistant” “Shared-Folder”
+* Optionally Start Home Assistant on Bootup of Synology
 
- - Install Home Assistant
- - Enable Home Assistant to be launched on [http://localhost:8123](http://localhost:8123)
 
-Using the Synology webadmin:
+Like you might have read above, at this time of writing, Synology has not provided a recent [Python3](https://www.synology.com/nl-nl/dsm/packages/py3k) package, so we have to create a recent Python3 package ourself.
+Do not worry though, were gonna try keep this as painless as possible.
 
- - Install python3 using the Synology Package Center
- - Create homeassistant user and add to the "users" group
+## {% linkable_title Preparing prerequisites %}
 
-SSH onto your synology & login as admin or root
+In this guide we are going to use a Linux based Operating System such as [Ubuntu](https://www.ubuntu.com/), [Mint](https://linuxmint.com/) or [Solus](https://getsol.us) to minimize difficulty. You will also need [Docker](https://docs.docker.com/install/).
+You can think of [Docker](https://docs.docker.com/install/) as a way more powerful [Chroot](https://help.ubuntu.com/community/BasicChroot) and containers as ready made mini operating systems.
 
- - Log in with your own administrator account
- - Switch to root using:
+---
+### {% linkable_title Installing Docker on [Mint, Ubuntu or derivatives](https://docs.docker.com/install/linux/docker-ce/ubuntu/) %}
 
+Refresh package lists:
 ```bash
-$ sudo -i
+# sudo apt-get update
 ```
+Install prerequisite packages which lets APT get packages over HTTPS:
+```bash
+# sudo apt-get install apt-transport-https ca-certificates curl software-properties-common
+```
+Add GPG key for the official Docker repository to your system:
+```bash
+# curl -fsSL https://download.docker.com/linux/ubuntu/gpg | sudo apt-key add -
+```
+<p class="note">
+The “lsb_release -cs” sub-command below returns the name of your Ubuntu distribution, such as xenial.
+Sometimes, in a distribution like Linux Mint, you might need to change “lsb_release -cs” to your parent Ubuntu distribution.
+For example, if you are using Linux Mint <b>Tara</b> (eg, 19), you could use <b>Bionic</b> (eg, 18.04).
+</p>
 
+```bash
+# sudo add-apt-repository "deb [arch=amd64] https://download.docker.com/linux/ubuntu $(lsb_release -cs) stable"
+```
+Refresh package lists for newly added Docker repository:
+```bash
+# sudo apt-get update
+```
+Install Docker:
+```bash
+# sudo apt-get install docker-ce
+```
+---
+### {% linkable_title Installing Docker on [Solus](https://getsol.us) %}
```

Comment:
Remove the Solus section and all parts that refer to Solus. If a user is using Solus it's usually an experienced user who an adapt the Ubuntu guide.